README.md

```diff
@@ -197,6 +197,48 @@ To see all of the available options, use the `--help` argument:
 npx lightningcss --help
 ```
 
+#### Browserslist configuration
+
+If no `--targets` option is provided, then `lightningcss` finds browserslist configuration,
+selects queries by environment and loads the resulting queries as targets.
+
+Pass the `--disable-browserslist` option if you do not want to load browserslist configuration or apply browserslist built-in defaults.
+
+Configuration resolution is modeled after the original `browserslist` nodeJS package.
+The configuration is resolved in the following order:
+
+- If a `BROWSERSLIST` environment variable is present, then load targets from its value. This is analog to the `--targets` CLI option.
+  _Example:_ `BROWSERSLIST="firefox ESR" lightningcss [OPTIONS] <INPUT_FILE>`
+- If a `BROWSERSLIST_CONFIG` environment variable is present, then resolve the file at the provided path.
+  Then parse and use targets from `package.json` or any browserslist configuration file pointed to by the environment variable.
+  _Example:_ `BROWSERSLIST_CONFIG="../config/browserslist" lightningcss [OPTIONS] <INPUT_FILE>`
+- If none of the above apply, then find, parse and use targets from the first `browserslist`, `package.json`
+  or `.browserslistrc` configuration file in any parent directory.
+
+Browserslist configuration files may contain sections denoted by angular brackets `[]`.
+Use these to specify different targets for different environments.
+Targets which are not placed in a section are added to `defaults` and used if no section applies matches the environment.
+
+_Example:_
+
+```
+# Defaults, applied when no other section matches the provided environment.
+firefox ESR
+
+[staging]
+# Targets applied only to the staging environment.
+samsung >= 4
+```
+
+When using parsed configuration from `browserslist`, `package.json` or `.browserslistrc` configuration files,
+the environment determined by
+
+- the `BROWSERSLIST_ENV` environment variable if present,
+- otherwise the `NODE_ENV` environment variable if present,
+- otherwise `production` is used.
+
+If no targets are found for the resulting environment, then the `defaults` configuration section is used.
```

Comment:
This part might be a breaking change. I think loading a browserlist file is ok to enable by default, but if a project had no browserslist configuration, then it might be weird if we suddenly started compiling differently. I think opting into downleveling, either via a `--targets` option or via a browserslist config file is the right way to go.

Reply:
I will check if browserslist-rs provides an API to disable defaults. As far as I remember, it is rather locked down, as `config::load()` is not public. In that case, the defaults alone could be opt-in (e.g. `--browserslist-defaults`?) while 

Otherwise, I'll turn it into `--browserslist` to keep the feature opt-in; avoiding the breaking change of applying browserslist default targets.

For a future lightningcss 2.0 we could reconsider enabling browserslist by default.

Reply:
I think `--targets "defaults"` should already cover that case, no?